### PR TITLE
Miawong/sc 78583/fix log in font size 

### PR DESCRIPTION
**Pull request by miaawong**
#### What this PR does / why we need it: Fix dynamicallyResizeText util to properly resize login text
When the appName is long: 
Screenshot 2023-06-02 at 9 39 04 AM

#### Which issue(s) this PR fixes:

Fixes # https://app.shortcut.com/replicated/story/78583/log-in-screen-text-is-too-small-now

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: web/src/features/Auth/components/SecureAdminConsole.tsx [hunk 1]
```diff
@@ -194,8 +194,7 @@ class SecureAdminConsole extends React.Component<Props, State> {
     const newFontSize = dynamicallyResizeText(
       this.loginText.current.innerHTML,
       this.loginText.current.clientWidth,
-      "32px",
-      "10px"
```
Comment (miaawong) [inline web/src/features/Auth/components/SecureAdminConsole.tsx:198]: remove the last argument, it was never being used properly

File: web/src/features/Auth/components/SecureAdminConsole.tsx [hunk 2]
```diff
@@ -231,9 +230,9 @@ class SecureAdminConsole extends React.Component<Props, State> {
     return null;
   };
 
-  componentDidUpdate(lastProps: Props) {
+  componentDidUpdate() {
     const { appName } = this.props;
-    if (appName && appName !== lastProps.appName) {
+    if (appName) {
```
Comment (miaawong) [inline web/src/features/Auth/components/SecureAdminConsole.tsx:235]: resize every time there's an appName

File: web/src/utilities/utilities.js [hunk 3]
```diff
@@ -133,42 +133,40 @@ export function secondsAgo(time) {
...
+export function dynamicallyResizeText(text, maxWidth, defaultFontSize) {
   let size;
   const resizerElm = document.createElement("p");
   resizerElm.textContent = text;
-  resizerElm.setAttribute("class", "u-fontWeight--bold");
-  resizerElm.setAttribute(
-    "style",
-    `visibility: hidden; z-index: -1; position: absolute; font-size: ${defaultFontSize}`
-  );
+  resizerElm.classList.add("u-fontWeight--bold");
```
Comment (miaawong) [inline web/src/utilities/utilities.js:140]: replace setAttribute to use classList or styles. (newer JavaScript API method)

File: web/src/utilities/utilities.js [hunk 4]
```diff
@@ -133,42 +133,40 @@ export function secondsAgo(time) {
...
+  const resizerWidth = () => resizerElm.getBoundingClientRect().width;
+  const resizerFontSize = () => parseInt(resizerElm.style.fontSize, 10);
+  if (resizerWidth() - maxWidth < 300) {
     resizerElm.remove();
-    return defaultFontSize;
+    size = defaultFontSize;
   }
 
-  while (resizerElm.getBoundingClientRect().width > maxWidth) {
-    if (resizerElm.style.fontSize > 10) {
```
Comment (miaawong) [inline web/src/utilities/utilities.js:158]: this check never ran because `resizerElm.style.fontSize` was actually a string

**Review by rianfowler — APPROVED** [review]
nice work

**Pull request merged**